packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/models/request_models/push_notification_data_request_model.dart

```diff
@@ -8,11 +8,12 @@ part 'push_notification_data_request_model.g.dart';
 class PushNotificationDataRequestModel {
   PushNotificationDataRequestModel(this.pushToken);
 
-  final String pushToken;
-
   factory PushNotificationDataRequestModel.fromJson(
           Map<String, dynamic> json) =>
       _$PushNotificationDataRequestModelFromJson(json);
+
+  final String pushToken;
```

Comment:
What's the point of moving this?

Reply:
Yesterday I got a hint from the analyzer to move the constructor up, but now it doesn't appear. Probably some hint from `rx_bloc_cli` got cached, since it has the `sort_constructors_first` option enabled... I'll return it back to its original state.